scripts/lua_def_file.py

```diff
@@ -0,0 +1,193 @@
+#!/bin/python3
```

Comment:
```suggestion
#!/usr/bin/python3
```
And why not make the script executable?

Reply:
No particular reason, I just looked at the other python script in that folder which isn't executable and I just followed.